### DISCUSSION TITLE
Welcome to Markdown-Previewer Discussions!

### DISCUSSION THREAD

**Discussion by hirentimbadiya** [hirentimbadiya/Markdown-Previewer#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Sudhin-star1** [comment]
Hello Hiren and community, I am a computer engineering undergrad and a total newbie to the world of Opensource. I am a web developer and a Machine Learning enthusiast. I am very excited to contribute to the OpenSource community.
